impeller/entity/contents/runtime_effect_contents.cc

```diff
@@ -165,7 +165,8 @@ bool RuntimeEffectContents::Render(const ContentContext& renderer,
   for (auto uniform : runtime_stage_->GetUniforms()) {
     // TODO(113715): Populate this metadata once GLES is able to handle
```

Comment:
This comment can now be removed.

Reply:
Keeping the comment - the `ShaderMetadata` is still not actually being filled with the data required to use runtime effects on GLES